### ISSUE TITLE
The npy file can not download

### ISSUE THREAD

**Issue by SilencerChen** [conan7882/GoogLeNet-Inception-tensorflow#3]
Thanks for you great job, it really help me.
But the net to download the pretrained GoogleNet npy file can not open. I don't know if it only happen on me. Anyway could you provide it for me?
Thanks so much.

**Comment by conan7882** [comment]
Did you put the correct path of the pre-trained npy file in 'setup_env.py'? 'PARA_DIR' is the path for the npy file.

**Comment by conan7882** [comment]
@SilencerChen  Sorry the link does not work now. I have updated the link for downloading.

**Comment by liukingly** [comment]
In the backpropagation process, how is the error of the last hidden layer connected by the Inception model calculated? Is the sum of the errors of the four branches?

**Comment by conan7882** [comment]
For training on CIFAR-10 the loss is the sum of two auxiliary classifiers and the final classifier.
